cvat-canvas/src/canvas.ts

```diff
@@ -3,81 +3,88 @@
 * SPDX-License-Identifier: MIT
 */
 
-/* eslint-disable */
-// Temporary disable eslint
+import { CanvasModel, CanvasModelImpl, Rotation } from './canvasModel';
+import { CanvasController, CanvasControllerImpl } from './canvasController';
```

Comment:
Is it possible to have separate files for an interface and its implementation? Just want to keep interface files really small.

Reply:
@nmanovic 
There aren't any problem to do that, but It is not common practice for typescript as far as i know. 
Files with all interfaces (also called typings) will be automatically generated by a typescript compiler. They have extensions d.ts.

Reply:
OK